### PR TITLE
UIDATIMP-714: Cover `<OptionsList>` component with tests

### PR DESCRIPTION
**Pull request by OleksandrHladchenko1**
## Purpose
Cover OptionsList component with tests

## Approach
Covered OptionsList component with tests

## Refs
https://issues.folio.org/browse/UIDATIMP-714

## Screenshots

image

### CODE REVIEW

**Review by IvanKryzhanovskyi — COMMENTED** [review]
Update Changelog file, please

File: src/components/OptionsList/OptionsList.test.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    const { getByText } = renderOptionsList(notEmptyOptionsList);
+
+    expect(getByText('name1')).toBeDefined();
+  });
+
+  it('should be rendered with an empty message', () => {
+    const { getByText } = renderOptionsList(emptyOptionsList);
+
+    expect(getByText('emptyMessage')).toBeDefined();
+  });
```
Comment (mariia-aloshyna) [inline src/components/OptionsList/OptionsList.test.js:79]: Add `describe` block for this case and the case above with description like `describe('when there is no options')` and then `it('then should be rendered with an empty message')`

File: src/components/OptionsList/OptionsList.test.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    expect(getByText('name1')).not.toBeVisible();
+
+    fireEvent.click(dropdownButton);
+
+    expect(getByText('name1')).toBeVisible();
+
+    fireEvent.click(dropdownButton);
+
+    expect(getByText('name1')).not.toBeVisible();
+  });
```
Comment (mariia-aloshyna) [inline src/components/OptionsList/OptionsList.test.js:105]: This test looks pretty overloaded to me, is it possible to split it into 2 test blocks? You can group it into `describe(`when clicking on dropdown`)` block e.g. and then check that after the first click the list is expanded and after the second click it is collapsed.

File: src/components/OptionsList/OptionsList.test.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+    expect(button).toBeDefined();
+
+    fireEvent.click(button);
+
+    expect(button).toBeDefined();
+  });
+
+  describe('when clicking on dropdown', () => {
+    it('should be shown by clicking on dropdown', () => {
```
Comment (mariia-aloshyna) [inline src/components/OptionsList/OptionsList.test.js:101]:
```suggestion
    it('dropdown options should appear', () => {
```

File: src/components/OptionsList/OptionsList.test.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      const dropdownButton = getByText('testLabel');
+
+      expect(getByText('name1')).not.toBeVisible();
+
+      fireEvent.click(dropdownButton);
+
+      expect(getByText('name1')).toBeVisible();
+    });
+
+    it('should be hidden by clicking on dropdown', () => {
```
Comment (mariia-aloshyna) [inline src/components/OptionsList/OptionsList.test.js:112]:
```suggestion
describe('when clicking on open dropdown) {
    it('dropdown options should be hidden', () => {
```

**Review by mariia-aloshyna — APPROVED** [review]

**Review by IvanKryzhanovskyi — APPROVED** [review]

**Pull request merged**